src/ec/packages/ec-component-tag/tag.html.twig

```diff
@@ -57,26 +61,28 @@
 
 {# Print the result #}
 
-{% if _tag.type != 'link' %}
-  <button class="{{ _css_class }}" type="button"{{ _extra_attributes|raw }}>
-    {{- _tag.label }}
-    {%- if _tag.type == 'removable' -%}
-      <span class="ecl-tag__icon">
-        {% include '../ec-component-icon/icon.html.twig' with {
-          icon: { type: 'ui', name: 'close', size: 'xs', path: default_icon_path },
-          extra_classes: 'ecl-tag__icon-close'
-        } only %}
-        {% include '../ec-component-icon/icon.html.twig' with {
-          icon: { type: 'ui', name: 'close-filled', size: 'xs', path: default_icon_path },
-          extra_classes: 'ecl-tag__icon-close-filled'
-        } only %}
-      </span>
-    {% endif -%}
-  </button>
-{% else %}
+{% if _tag.type == 'link' %}
   <a href="{{ _tag.path }}" class="{{ _css_class }}"{{ _extra_attributes|raw }}>
     {{- _tag.label -}}
   </a>
+{% elseif _tag.type == 'button' %}
+  <button class="{{ _css_class }}" type="button"{{ _extra_attributes|raw }}>
```

Comment:
i know this might be inconsistent in all the templates, but why not starting fixing these inconsistencies? :) here i don't get the reason why we put a space between the classes and the type but not between the type and the extra_attributes. Both are correct, but it's weird to have this inconsistency, imho ;)